sfm_pc/management/commands/import_google_doc.py

```diff
@@ -420,26 +289,36 @@ def get_sheets_from_folder(self, folder):
         entities = (
             ('organization', 'units', 'orgs'),
             ('person', 'persons', 'persons'),
-            ('person_extra', 'persons_extra', 'persons_extra'),
             ('event', 'incidents', 'events'),
-            ('source', 'sources', 'values')
+            ('person_extra', 'persons_extra', 'persons_extra')
         )
         for entity_type, filename, title in entities:
             path = os.path.join(folder, filename + '.csv')
-            if not os.path.isfile(path):
-                raise OSError('Required file {path} not found.'.format(path=path))
 
-            with open(path) as fobj:
-                reader = csv.reader(fobj)
-                # We could use csv.DictReader directly here, but instead use
-                # the format_dict_reader convenience method to test it
-                records = self.format_dict_reader(list(reader))
+            records = self.get_records_from_csv(path)
```

Comment:
moved this logic into a function because we're calling it twice. since sources isn't living in each country's directory, we can include it in the `entities` tuple and instead have to call it by hand on line 303.

Reply:
i just realized a way we might could include the sources in the entities tuple — just go ahead and build up the filename in the tuple